js/unhide-nav.js

```diff
@@ -0,0 +1,22 @@
+/** Unhide the nav for screen readers when clicked **/
+
+document.addEventListener('DOMContentLoaded', function() {
```

Comment:
Suggestion for better accessibility: the 'menu' button counts as an expanding element on mobile; since you are already handing some other additions based on it's expanded state, I'd recommend adding `aria-expanded="false"`, and swapping to `true` when the menu expands. Make sure you also add `aria-controls="(id_of_the_menu)" ` as well. See https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded

Reply:
I'll work on adding this!